Dockerfile

```diff
@@ -1,9 +1,9 @@
 FROM  node:latest
-RUN   mkdir -p /root/app
-WORKDIR  /root/app
-COPY    package.json  /root/app/package.json
+RUN   mkdir -p /app
+WORKDIR  /app
+COPY    package.json /app
```

Comment:
No need to specify `/app` since your workdir context is already inside that dir,.You can use `.` (current dir context) here.